plugins/module_utils/pacemaker.py

```diff
@@ -7,7 +7,7 @@
 __metaclass__ = type
 
 from ansible_collections.community.general.plugins.module_utils.cmd_runner import CmdRunner, cmd_runner_fmt
-
+import re
```

Comment:
nit picking - as a general convention/recommendation in Python, put the imports from the standard library above the other ones.